datm/datm_datamode_cplhist_mod.F90

```diff
@@ -0,0 +1,233 @@
+module datm_datamode_cplhist_mod
+
+  use ESMF
```

Comment:
Please add an only clause here.

Reply:
@jedwards4b - This should be done for ALL of the modules not just this one. Can you please raise this as an issue - and we could do this all at once for all of the data components - not just this one. If this is then your only reservation for this PR can you please accept it once you have raised the issue.